web/controllers/notes_controller.ex

```diff
@@ -0,0 +1,8 @@
+defmodule Notox.NoteController do
+  use Notox.Web, :controller
+
+  def index(conn, _params) do
+    notes = Repo.all(Notox.Note)
+    render conn, "index.html", notes: notes
```

Comment:
You can also do `render conn, :index, notes: notes`. Then if you ever want to render `json` you don't have to change this line.